server/api/offline_batch_results.py

```diff
@@ -185,6 +185,89 @@ def record_offline_batch_results(
     return jsonify(status="ok")
 
 
+@api.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/round/<round_id>/results/batch/",
```

Comment:
im gonna let this trailing slash slide this time ;)

Reply:
lol can't get it past you.